clar2wasm/tests/wasm-generation/sequences.rs

```diff
@@ -114,6 +114,59 @@ proptest! {
     }
 }
 
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_map_not(
+        seq in (0usize..=20).prop_flat_map(PropValue::any_sequence),
+        val in any::<u32>()
+    ) {
+        let expected = !extract_sequence(seq.clone()).len() > val as usize;
+        let snippet = format!("(map not (list (> (len {seq}) u{val})))");
+
+        crosscheck(
+            &snippet,
+            Ok(Some(
+                Value::Sequence(
+                    SequenceData::List(
+                        ListData {
+                            data: vec![Value::Bool(expected)],
+                            type_signature: ListTypeData::new_list(TypeSignature::BoolType, 1).unwrap()
+                        }
+                    )
+                )
+            ))
+        )
+    }
+}
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_map_add(
+        (seq_1, seq_2) in (0usize..=20).prop_flat_map(|v| (PropValue::any_sequence(v), PropValue::any_sequence(v)))
+    ) {
+        let expected = extract_sequence(seq_1.clone()).len() + extract_sequence(seq_2.clone()).len();
+        let snippet = format!("(map + (list (len {seq_1})) (list (len {seq_2})))");
```

Comment:
That check was naive. Changed for an implementation that uses concat to use memory allocation.

Reply:
I don't think it was naive. A test for variadic functions support sounds like a decent idea.
Both of those examples should work: 

- `(map + (list 1 2) (list 2 3))`
- `(map + (list 1 2) (list 2 3) (list 4 5))`

It would be nice to have a test that covers it.